.github/workflows/connector_nightly_builds_dagger.yml

```diff
@@ -1,17 +1,17 @@
 name: POC Connectors CI - nightly builds
 
 on:
-  # schedule:
-  #   # 11AM UTC is 12AM CET, 4AM PST.
-  #   - cron: "0 8 * * *"
+  schedule:
+    # 11AM UTC is 12AM CET, 4AM PST.
+    - cron: "0 8 * * *"
   workflow_dispatch:
     inputs:
       runs-on:
         type: string
         default: dev-large-runner
         required: true
       test-connectors-options:
-        default: --concurrency=5 --release-stage=generally_available --release-stage=beta
+        default: --concurrency=5 --release-stage=generally_available
```

Comment:
@alafanechere I was noticing some flakyness even when only running GA connectors.

Im thinking we start the nightlies off with GA, then if they match the current nightlies we expand to beta